@babel/runtime dep missing for top-level mjml package

**Describe the bug**

(this is a regression from 4.6.2)

In 4.6.3 the npm module lib/index.js for `mjml` contains

```
var _interopRequireWildcard = require("@babel/runtime/helpers/interopRequireWildcard");
```

however, @babel/runtime is not a dependency for `mjml` (it is for its other projects like mjml-accordian)

This means that when you install without an explicit @babel/runtime in your package.json, a require of mjml produces

```
Uncaught Error: Cannot find module '@babel/runtime/helpers/interopRequireWildcard'
Require stack:
- /app/node_modules/mjml/lib/index.js
- <repl>
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:957:15)
    at Function.Module._load (internal/modules/cjs/loader.js:840:27)
    at Module.require (internal/modules/cjs/loader.js:1019:19)
    at require (internal/modules/cjs/helpers.js:77:18) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/app/node_modules/mjml/lib/index.js', '<repl>' ]
}
```

Can we add this to the main `mjml` package as a dependency?

**MJML environment (please complete the following information):**
 - OS: MacOS, linux
 - MJML Version 4.5.3

Can you provide more informations like node version, did you tried a clean install ...

> On 25 Sep 2020, at 18:53, Zygimantas Berziunas <notifications@github.com> wrote:
> 
> ﻿
> After updating to 4.7 I started getting EXEC : error : Cannot find module '@babel/runtime/helpers/createSuper'
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or unsubscribe.

--- a/packages/mjml/package.json
+++ b/packages/mjml/package.json
@@ -26,6 +26,7 @@
     "test": "node ./test/test-html-attributes.js"
   },
   "dependencies": {
+    "@babel/runtime": "^7.8.7",
     "mjml-accordion": "4.7.1",
     "mjml-body": "4.7.1",
     "mjml-button": "4.7.1",

